modules/light-clients/08-wasm/types/client_state_test.go

```diff
@@ -86,81 +92,66 @@ func (suite *TypesTestSuite) TestStatusGrandpa() {
 	}
 }
 
-// func (suite *TypesTestSuite) TestStatusTendermint() {
-// 	var (
-// 		path          *ibctesting.Path
-// 		clientState   *types.ClientState
-// 		tmClientState *tmtypes.ClientState
-// 	)
-
-// 	testCases := []struct {
-// 		name      string
-// 		malleate  func()
-// 		expStatus exported.Status
-// 	}{
-// 		{
-// 			"client is active",
-// 			func() {},
-// 			exported.Active,
-// 		},
-// 		{
-// 			"client is frozen",
-// 			func() {
-// 				tmClientState.FrozenHeight = clienttypes.NewHeight(0, 1)
-
-// 				wasmData, err := suite.chainA.Codec.MarshalInterface(tmClientState)
-// 				suite.Require().NoError(err)
-
-// 				clientState.Data = wasmData
-// 				path.EndpointA.SetClientState(clientState)
-// 			},
-// 			exported.Frozen,
-// 		},
-// 		{
-// 			"client status without consensus state",
-// 			func() {
-// 				latestHeight := clientState.LatestHeight.Increment().(clienttypes.Height)
-// 				tmClientState.LatestHeight = latestHeight
-
-// 				wasmData, err := suite.chainA.Codec.MarshalInterface(tmClientState)
-// 				suite.Require().NoError(err)
-
-// 				clientState.Data = wasmData
-// 				clientState.LatestHeight = latestHeight
-// 				path.EndpointA.SetClientState(clientState)
-// 			},
-// 			exported.Expired,
-// 		},
-// 		{
-// 			"client status is expired",
-// 			func() {
-// 				suite.coordinator.IncrementTimeBy(tmClientState.TrustingPeriod)
-// 			},
-// 			exported.Expired,
-// 		},
-// 	}
+func (suite *TypesTestSuite) TestStatus() {
+	testCases := []struct {
+		name      string
+		malleate  func()
+		expStatus exported.Status
+	}{
+		{
+			"client is active",
+			func() {},
+			exported.Active,
+		},
+		{
+			"client is frozen",
+			func() {
+				suite.mockVM.QueryFn = func(codeID wasmvm.Checksum, env wasmvmtypes.Env, queryMsg []byte, store wasmvm.KVStore, goapi wasmvm.GoAPI, querier wasmvm.Querier, gasMeter wasmvm.GasMeter, gasLimit uint64, deserCost wasmvmtypes.UFraction) ([]byte, uint64, error) {
```

Comment:
this is super nice, one thing that might be useful for readability is a function alias, but editors will auto fill this stuff for you so I don't feel super strongly about it.

Reply:
I don't feel strongly either, so I'll let it be a followup if we so chose